### ISSUE TITLE
default view of composes should be reversed in order

### ISSUE THREAD

**Issue by ausil** [product-definition-center/product-definition-center#435]
if you go to https://pdc.fedoraproject.org/compose/ the composes are sorted oldest first, typically when you go to that page you are looking for a recent compose. so you have to go straight to the end and page back. the default view should be the most recent composes first

**Comment by bliuredhat** [comment]
@ausil , yeah, and I think it's easy to fix as your comments. The current status is ordered by id, the solution is to reverse the rule.

**Comment by bliuredhat** [comment]
the patch for this issue: https://github.com/product-definition-center/product-definition-center/pull/436

**Comment by sochotnicky** [comment]
This should not be merged - it breaks the API compatibility. Unless you want to start v2 API

**Comment by bliuredhat** [comment]
👍 , Got it.

**Comment by bliuredhat** [comment]
this issue have been solved and merged the patch to master branch. It will be verified in new version.
